app/App_Resources/Android/src/main/AndroidManifest.xml

```diff
@@ -36,5 +36,9 @@
 			</intent-filter>
 		</activity>
 		<activity android:name="com.tns.ErrorReportActivity"/>
+		<service
```

Comment:
We can probably add this part in "src/release/" in the Android Runtime template. The downside is that if the user uses a foreground service himself he might realize he is missing the permission in release:
```
<uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
```